x-pack/test_serverless/functional/test_suites/security/ftr/cases/configure.ts

```diff
@@ -26,8 +26,18 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
       await svlCommonPage.login();
       await svlSecNavigation.navigateToLandingPage();
       await testSubjects.click('solutionSideNavItemLink-cases');
+      await header.waitUntilLoadingHasFinished();
+
+      await retry.waitFor('configure-case-button exist', async () => {
```

Comment:
maybe I'm confused but isn't this the same as
`await retry.waitFor('configure-case-button exist', testSubjects.exists('configure-case-button'));`
?

Reply:
Probably yes 🙂. I never thought about it tbh.